app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -155,6 +163,7 @@ class RecentlyPlayedFragmentTest {
     ).use {
       testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.recently_played_toolbar)).perform(click())
+      assertThat(activityScenarioRule.scenario.state).isEqualTo(Lifecycle.State.DESTROYED)
```

Comment:
Can we do this using `activityScenarioRule`?

Use Reference from `testStoryFragment_clickOnToolbarNavigationButton_closeActivity`

Reply:
@anandwana001 I don't find any method using activityScenarioRule which can ensure the activity is finished or not.

Reply:
Check this method if it helps as reference https://github.com/oppia/oppia-android/blob/eb137fba7718bcf5b4eea5c379d727e88166607f/app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt#L131

Reply:
I have also checked this. In this test we are using activityTestRule to check the activity is finished. using activityScenarioRule i am able to get the activity using activityScenarioRule.javaClass but not able to find any method which can ensure the activity is finished or not

Reply:
Using ActivityScenarioRule, I think this is the only way to obtain the state of an activity. 
https://github.com/android/android-test/issues/235